Makefile

```diff
@@ -248,15 +248,39 @@ tools/custom-gcl: tools/structwrite .custom-gcl.yml
 	golangci-lint custom
 
 .PHONY: lint
-lint: tidy tools/custom-gcl
+lint: tools/custom-gcl
```

Comment:
removing `tidy` here because agents always get confused when it fails due to unstaged changes. since `lint` is not used in CI, I don't think should should cause any issues.